core/templates/domain/question/question-backend-api.service.spec.ts

```diff
@@ -153,13 +153,13 @@ describe('Question backend Api service', () => {
       let questionCountHandlerUrl = (
         '/questions_count_handler/' + encodeURIComponent(1));
 
-      questionBackendApiService.fetchQuestionCountBySkillIds(
+      questionBackendApiService.fetchQuestionCountForSkillIds(
```

Comment:
Include the word "total" in the method name.

Please check all the other similarly named methods in this PR (backend and frontend) and ensure they include the word "total", too. This word is important.

Reply:
Done.